fuseki/sample-ubd.trig

```diff
@@ -197,6 +198,11 @@ graph <cube-project/ubd/cube-data> {
     <https://environment.ld.admin.ch/foen/ubd/28/unit>            <http://qudt.org/vocab/unit/MicroGM-PER-M3> ;
   .
 
+  <https://environment.ld.admin.ch/foen/ubd/28/station/blBAS>
+    schema:identifier "blBAS" ;
+    vaem:owner <https://environment.ld.admin.ch/foen/ubd/28/maintainer/blBAS> ;
```

Comment:
This is not mapped in transformation but I added so that we can test that only objects of observation dimensions are considered for the `schema:sameAs` treatment